library/common/types/c_types.h

```diff
@@ -103,6 +103,14 @@ extern "C" { // utility functions
  */
 void* safe_malloc(size_t size);
 
+/**
+ * calloc wrapper that asserts that the returned pointer is valid. Otherwise, the program exits.
```

Comment:
This isn't necessarily true since we compile out release asserts, right? Maybe worth clarifying

Reply:
It's a little confusing, but actually the other way around. `RELEASE_ASSERT` is never compiled out, regular `ASSERT` can be.